src/main/java/seedu/duke/Duke.java

```diff
@@ -28,7 +28,7 @@
     private TaskList taskList;
     private LessonList lessonList;
     private ModuleList moduleList;
-    private FullModuleList fullModuleList;
+    public static FullModuleList fullModuleList;
```

Comment:
Hmm.. any reason for changing it to `public static`?

Reply:
The only other option is to pass it as another parameter to `command.execute()`, but I felt that making it public static made more sense since it's a reference for `AddModuleCommand`, not something that needs to be modified. If you have any other suggestions I can try them out though.